components/desktop/libreoffice/Makefile

```diff
@@ -487,12 +485,12 @@ LIBREOFFICE_SHARE_DIR = /usr/lib/amd64/libreoffice/share
 
 NSS_LIB_DIR=/usr/lib/mps/amd64
 
-JAVA17_LIB_DIR=/usr/jdk/openjdk17/lib
+JAVA21_LIB_DIR=/usr/jdk/openjdk21/lib
```

Comment:
I suggest to define something like `JAVA_LIB_DIR = $(JAVA_HOME)/lib` here instead of `JAVA21_LIB_DIR`.  This will make sure `libreoffice` always builds with the latest LTS java version without further changes needed here.  If you would need (or want) different (a non-default) java version then simple addition of `JAVA_VERSION = 11` or similar would do the trick.